code/modules/clothing/suits/jobs.dm

```diff
@@ -151,6 +156,23 @@
 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
 	allowed = list(/obj/item/kitchen/knife)
 
+//Chief Engineer
+/obj/item/clothing/suit/mantle/chief_engineer
+	name = "chief engineer's mantle"
+	desc = "A sturdy, protective poncho designed for the Chief Engineer."
+	icon_state = "cemantle"
+	item_state = "cemantle"
+	armor = list(melee = 25, bullet = 20, laser = 25, energy = 10, bomb = 20, bio = 0, rad = 20)

```

Comment:
CE doesn't have anything other than his space suit that gives him armor. 

This is just a straight up buff for the CE since it doens't slow him down. Please remove this.